tests/functional/backward_compatibility/test_versions.py

```diff
@@ -99,11 +99,13 @@ def get_virtualenv_dir(version):
 
 def fill_from_cmdline_api(orion_script, version):
     """Add experiments and trials using the commandline API"""
+    # TODO: Adapt when removing init_only after deprecation phase
```

Comment:
We might want to add a version target for these TODO so they don't get lost in time and are easy to reference from an action item.